code/game/gamemodes/miniantags/slaughter/slaughter.dm

```diff
@@ -247,10 +247,7 @@
 	user.visible_message("<span class='warning'>[user] raises [src] to [user.p_their()] mouth and tears into it with [user.p_their()] teeth!</span>", \
 						 "<span class='danger'>An unnatural hunger consumes you. You raise [src] to your mouth and devour it!</span>")
 	playsound(user, 'sound/misc/demon_consume.ogg', 50, 1)
-	for(var/obj/effect/proc_holder/spell/knownspell in user.mind.spell_list)
-		if(knownspell.type == /obj/effect/proc_holder/spell/bloodcrawl)
-			qdel(src)
-			return
```

Comment:
These lines shouldn't have been removed completely, only the return should have been. I tested this out, and without these lines, when someone eats a second heart to get `BLOODCRAWL_EAT`, the heart doesn't get deleted, and it instead gets dropped on the floor. Which means another person can pick up the heart and use it to get normal `BLOODCRAWL`.

Reply:
Alright, weirdly enough during my initial testing removal of these lines actually got some of the code actually working, I'll fix this then